src/refactorings/simplify-ternary/index.ts

```diff
@@ -10,7 +14,12 @@ const config: RefactoringWithActionProvider = {
   },
   actionProvider: {
     message: "Simplify ternary",
-    canPerform: hasTernaryToSimplify
+    createVisitor: createTernaryToSimplifyVisitor,
+    updateMessage(path: t.NodePath<any>): void {
+      this.message = `Simplify ternary (for demo purposes only: ${
```

Comment:
Nice 👍 

Don't forget to drop it before we merge, since it's only for demoing it can be done.

Reply:
Already removed in one of the other PRs. :)